File: problem_solving/uva/414.py
inf=float('inf')
while True:
	n = int(input())
	if n==0: break
	a=[]
	sp=[]
	max_count=inf
	for i in range(n):
		x=input()
		a.append(x)
		cnt = x.count(' ')
		sp.append(cnt)
		max_count=min(max_count,cnt)
	
	if n==1:
		print(0)
		continue
	
	s=0
	for i in sp:
		s+=abs(max_count-i)
	
	print(s)
	

File: problem_solving/uva/414.cpp
#include <bits/stdc++.h>
using namespace std;

int N,mn;
string s;
vector<int> cnt;

inline void solve() {
	int sm=0;
	for(int&i:cnt)sm+=abs(i-mn);
	cout<<sm<<'\n';
}

void read() {
	while (cin >> N && N > 0) {
		cnt.resize(N);
		mn=INT_MAX;
		for(int i=0;i<N;++i) {
			getline(cin,s);
			cnt[i]=count(s.begin(),s.end(),' ');
			cout<<cnt[i]<<endl;
			mn=min(mn,cnt[i]);
			cout<<mn<<endl;
		}
		if(N==1){
			cout<<0<<'\n';
			continue;
		}
		solve();
	}
}

int main() {
	ios::sync_with_stdio(0);
	cin.tie(0);
	
	read();
	
	return 0;
}

/* 
4
XXXX                XXXXX
XXX               XXXXXXX
XXXXX                XXXX
XX                 XXXXXX
0
*/
